AK/Time.h

```diff
@@ -41,6 +42,12 @@ constexpr bool is_leap_year(int year)
     return year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);
 }
 
+#ifndef KERNEL
+Optional<struct tm> convert_formatted_string_to_timespec(StringView input, StringView format, size_t&, size_t&);
+Optional<struct tm> convert_formatted_string_to_timespec(StringView input, StringView format, size_t&);
+Optional<struct tm> convert_formatted_string_to_timespec_restrictively(StringView input, StringView format);
```

Comment:
If you add a `using` declaration at the bottom of the file (just like the other methods do), then you can drop the `AK::` qualification in other places.